condor/condor_meter

```diff
@@ -711,7 +710,19 @@ def classadToJUR(classad):
 
     setIfExists(r.Queue, classad, "JobUniverse", "Condor's JobUniverse field", setstr=True)
     setIfExists(r.NodeCount, classad, 'MaxHosts', "max")
-
+    ########################################################################################
+    ########################################################################################
+    # Code added to send to arbitrary Ads SOFTWARE-2714
+    ArbitraryJobAds = GratiaCore.Config.getConfigAttribute("ArbitraryJobAdAttributes")
+    DebugPrint(5, "ArbitraryJobAd: %s" % str(ArbitraryJobAds))
+    Arbirtray_adlist = ArbitraryJobAds.split(",")
+    DebugPrint(5, "Going through list: %s " % Arbirtray_adlist)
+    for arbitaryAd in  Arbirtray_adlist:
+        if arbitaryAd in classad:
+            DebugPrint(5, "Arbitrary ad: %s found in classAd" % arbitaryAd) 
+            DebugPrint(5, "Value of arbitrary classAd: %s" % str(classad[str(arbitaryAd)]))
+            r.AdditionalInfo(arbitaryAd, str(classad[str(arbitaryAd)]))
```

Comment:
For complex expressions, this will simply convert the expression to a string instead of evaluate the expression.

I think we want to evaluate the expression.